R/queue_didehpc.R

```diff
@@ -196,6 +198,7 @@ queue_didehpc_ <- R6::R6Class(
     ##'
     ##' @param task_ids Vector of task identifiers to look up
     dide_id = function(task_ids) {
+      if (length(task_ids) == 0) return(c())
```

Comment:
```suggestion
      if (length(task_ids) == 0) {
        return(NULL)
      }
```

style, if you don't mind